components/RevisionTimeline/RevisionTimeline.tsx

```diff
@@ -0,0 +1,42 @@
+import { format } from 'date-fns';
+import { Submission } from 'data/flexibleForms/forms.types';
+
+interface Props {
+  submission: Submission;
+}
+
+const RevisionTimeline = ({ submission }: Props): React.ReactElement | null => {
+  // reverse the array so it's in reverse-chronological order and take the three most recent events
+  const revisions = submission.editHistory.concat([]).reverse().slice(0, 3);
+
+  return (
+    <>
+      <h2>History</h2>
+      <ol className="lbh-timeline">
+        {revisions.map((revision, i) => (
+          <li
+            className={`lbh-timeline__event lbh-timeline__event--minor ${
+              i === revisions.length - 1 && 'lbh-timeline__event--gap-below'
```

Comment:
Would a CSS psuedoselector to select the last element make more sense here

Reply:
all the pseudoselectors are used up! for other things! most of the timeline is actually styled pseudoselectors